librz/syscall/syscall.c

```diff
@@ -377,16 +377,19 @@ RZ_API RzSyscallItem *rz_syscall_get(RzSyscall *s, int num, int swi) {
 	return rz_syscall_item_new_from_string(ret, ret2);
 }
 
-RZ_API int rz_syscall_get_num(RzSyscall *s, const char *str) {
-	rz_return_val_if_fail(s && str, -1);
+RZ_API bool rz_syscall_get_num(RzSyscall *s, const char *str, int *num) {
```

Comment:
```suggestion
RZ_API bool rz_syscall_get_num(RZ_NONNULL RZ_BORROW RzSyscall *s, RZ_NONNULL const char *str, int *num) {
```

Reply:
Please document the function as well. Since you know now how it works and what it does.